src/Components/PaperEditor/TranscriptsContainer/LabelsList/index.js

```diff
@@ -124,62 +121,45 @@ const LabelsList = (props) => {
       const { label, color, id, description } = l;
 
       return (
-        <ListGroup.Item style={ { width: '100%' } } key={ cuid() }>
+        <ListGroup.Item
+          key={ cuid() }
+          className='LabelsList__list-item'
+          href='#'
+          style= { { backgroundColor: 'white', borderColor: 'rgba(0, 0, 0, 0.125)' } }>
           <Row>
-            <Col
-              xs={ 1 }
-              sm={ 1 }
-              md={ 1 }
-              lg={ 1 }
-              xl={ 1 }
-              style={ { backgroundColor: color } }
-              title={ label }
-            ></Col>
-            <Col xs={ 6 } sm={ 6 } md={ 6 } lg={ 6 } xl={ 6 } title={ label }>
-              {label}
-            </Col>
+            <Button
```

Comment:
it looks like there is some behaviour here inherited from bootstrap that makes the text disappear (turn white) when hovering on `LabelsList__label-color-square` span, that won't get overwritten by the `:hover` style in the stylesheet. 

I think turning this to regular HTML button should do the job.